OpenSim/Region/CoreModules/ServiceConnectors/Interregion/RESTInterregionComms.cs

```diff
@@ -91,9 +91,7 @@ public virtual void Initialize(Scene scene, IConfigSource config)
 
 
                 IConfig startupConfig = config.Configs["Communications"];
-                if ((startupConfig == null)
-                    || (startupConfig != null)
-                    && (startupConfig.GetString("InterregionComms", "RESTComms") == "RESTComms"))
+                if (startupConfig != null && startupConfig.GetString("InterregionComms", "RESTComms") == "RESTComms")
                 {
```

Comment:
while the second startupConfig != null test was previously redundant (because of the || ), this commit changes the semantics for if there's no Communications section at all, from defaulting to using this module to not using it.  That seems like it is the improved behavior; I'm just noting the semantic change (which should be okay here -- I don't think we'll have regions that _don't_ have a Communications section and _do_ want RESTComms enabled. As a bit of an aside though, I do wish code like this checking for specific names was case-insensitive.